### PR TITLE
6lowpan: some border router fixes

### PR DESCRIPTION
**Pull request by OlegHahm**
* disable sending of router solicitations on the 6lo interface for border routers
* use the new address as source address for router advertisements if a new address has been configured

### CODE REVIEW

File: sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c [hunk 1]
```diff
@@ -109,16 +109,19 @@ static ipv6_addr_t *_add_addr_to_entry(gnrc_ipv6_netif_t *entry, const ipv6_addr
...
                 /* New prefixes MAY allow the router to retransmit up to
                  * GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF unsolicited RA
                  * (see https://tools.ietf.org/html/rfc4861#section-6.2.4) */
-                entry->rtr_adv_count = GNRC_NDP_MAX_INIT_RTR_ADV_NUMOF;
-                gnrc_ndp_router_retrans_rtr_adv(entry);
-#elif defined(MODULE_GNRC_SIXLOWPAN_ND_ROUTER)
-                if (entry->flags & GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN) {
-                    gnrc_ndp_internal_send_rtr_adv(entry->pid, &tmp_addr->addr,
-                                                   NULL, false);
+                else {
```
Comment (authmillenon) [inline sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c]: This would break normal IPv6 brouters, since the `else` has no corresponding `if` there.

Comment (OlegHahm) [inline sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c]: addressed

**Comment by authmillenon** [comment]
ACK. Please squash.

**Comment by OlegHahm** [comment]
squashed

**Pull request merged**